c/VerifyThis.set

```diff
@@ -0,0 +1 @@
+verifythis/*.yml
```

Comment:
Can these files be added to an existing category? If not, they should probably be at least in  a subcategory of an existing category, right?

Reply:
ReachSafety-Recursive.setPlaced into ReachSafety-Loops.set and